Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -373,13 +409,19 @@ class BrowserViewController: UIViewController {
     }
 
     private func setupConstraints() {
+        topTabsContainer.snp_makeConstraints { make in
+            make.left.right.equalTo(self.header)
+            make.top.equalTo(urlBarTopTabsContainer)
+        }
+        
         urlBar.snp_makeConstraints { make in
-            make.edges.equalTo(self.header)
+            make.left.right.bottom.equalTo(urlBarTopTabsContainer)
```

Comment:
Not sure if it will effect things but we should use `leading` and `trailing` for better RTL support.